Lib/configparser.py

```diff
@@ -157,6 +157,7 @@
            "LegacyInterpolation", "SectionProxy", "ConverterMapping",
            "DEFAULTSECT", "MAX_INTERPOLATION_DEPTH"]
 
+DEFAULT_DICT = dict
```

Comment:
What is the purpose of introducing this variable?

Reply:
The unit tests reach into configparser to use the default dict implementation, and I was unsure of whether it would be best practice to continue that behavior or just make the unit tests reference dict directly.

Reply:
Rename this back to `_default_dict` to keep the previous API.